src/EnergyPlus/DataHVACGlobals.cc

```diff
@@ -166,6 +165,10 @@ namespace DataHVACGlobals {
 	int const Coil_CoolingAirToAirVariableSpeed( 28 );
 	int const Coil_HeatingAirToAirVariableSpeed( 29 );
 	int const CoilDX_HeatPumpWaterHeaterVariableSpeed( 30 );
+	int const CoilDX_HeatPumpWaterHeaterWrapped( 31 );
+	
+	int const CoilVRF_FluidTCtrl_Cooling( 32 );
+	int const CoilVRF_FluidTCtrl_Heating( 33 );
```

Comment:
I successfully merged the develop branch into mine several days ago. In the merge before that, about 15 days ago, there were merge conflicts and I modified my codes to avoid the conflicts. Anyway, I will be very careful when merge.